python/pyspark/ml/regression.py

```diff
@@ -738,11 +744,29 @@ class TreeEnsembleModels(JavaModel):
     """
 
     @property
+    @since("2.0.0")
+    def trees(self):
```

Comment:
```GBTClassifier.trees``` should return an array of ```DecisionTreeRegressionModel```, so it does not necessary to override this method. This is what I really mean.